### ISSUE TITLE
IllegalArgumentException when removing OnFrameMetricsAvailableListener

### ISSUE THREAD

**Issue by tevjef** [DataDog/dd-sdk-android#1875]
### Stack trace

```
Caused by java.lang.IllegalArgumentException: attempt to remove OnFrameMetricsAvailableListener that was never added
       at android.view.View.removeFrameMetricsListener(View.java:7948)
       at android.view.Window.removeOnFrameMetricsAvailableListener(Window.java:997)
       at com.datadog.android.rum.internal.vitals.JankStatsActivityLifecycleListener.unregisterMetricListener(JankStatsActivityLifecycleListener.kt:216)
       at com.datadog.android.rum.internal.vitals.JankStatsActivityLifecycleListener.onActivityDestroyed(JankStatsActivityLifecycleListener.kt:163)
       at android.app.Application.dispatchActivityDestroyed(Application.java:576)
       at android.app.Activity.dispatchActivityDestroyed(Activity.java:1647)
       at android.app.Activity.onDestroy(Activity.java:2851)
       at androidx.fragment.app.FragmentActivity.onDestroy(FragmentActivity.java:257)
       at androidx.appcompat.app.AppCompatActivity.onDestroy(AppCompatActivity.java:283)
       at com.company.features.onboarding.signup.main.ui.activity.Hilt_OnboardingSignupActivity.onDestroy(Hilt_OnboardingSignupActivity.java:64)
       at com.company.features.onboarding.signup.main.ui.activity.OnboardingSignupActivity.onDestroy(OnboardingSignupActivity.kt:77)
       at android.app.Activity.performDestroy(Activity.java:9236)
       at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1491)
       at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:6135)
       at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:6180)
       at android.app.servertransaction.DestroyActivityItem.execute(DestroyActivityItem.java:47)
       at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
       at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:180)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:98)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2685)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loopOnce(Looper.java:230)
       at android.os.Looper.loop(Looper.java:319)
       at android.app.ActivityThread.main(ActivityThread.java:8893)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:608)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
```

### Volume

Not yet significant

### Affected SDK versions

2.6.0

### Latest working SDK version

2.6.0

### Does the crash manifest in the latest SDK version?

Yes

### Kotlin / Java version

1.9.22

### Gradle / AGP version

8.6

### Device Information

Brand:Samsung
Model:Galaxy S22+
Orientation:Portrait
RAM free: 3.23 GB
Disk free: 92.53 GB
Operating System
Version:Android 14
Orientation:Portrait
Rooted:No

**Comment by xgouchet** [comment]
Hi @tevjef , thanks for opening this ticket, this is an issue we're already aware of and writing a fix as we speak. We should have a new version out very soon.

**Comment by xgouchet** [comment]
Hey, the fix should be available in version `2.6.2`

**Comment by 0xnm** [comment]
I'm closing this issue since the fix was released in version `2.6.2`.

**Comment by MattFoley** [comment]
Thanks guys! Just a heads up that it did affect 2.7.1 as well. I'll switch to 2.6.2 for now!
